### PR TITLE
Issue 837

### PR DESCRIPTION
**Pull request by LindaMcNulty**
Removal of Galen's "De Humero iis..." from the middle of his commentary on Hippocrates, plus creation of a separate xml file and proper __cts__ file for tlg0057.tlg096. Also updated lang tags, TEI headers, and document structure in all files affected by this issue.

### CODE REVIEW

File: data/tlg0057/tlg096/tlg0057.tlg096.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,1167 @@
...
+		<encodingDesc>
+			<p>The following text is encoded in accordance with EpiDoc standards and with the CTS/CITE
+				Architecture.</p>
+			  <refsDecl n="CTS">
+				<cRefPattern matchPattern="(.+)" n="chapter" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
+			</refsDecl>
+		</encodingDesc>
+		<profileDesc>
+			<langUsage>
+				<language ident="greek">Greek</language>
```
Comment (sonofmun) [inline data/tlg0057/tlg096/tlg0057.tlg096.1st1K-grc1.xml]: ident here should be "grc" instead of "greek"

**Comment by LindaMcNulty** [comment]
Okay, all set!

**Comment by sonofmun** [comment]
@LindaMcNulty You need to make the same change to the ident attribute on tlg0057.tlg095.1st1K-grc1.xml as well.

**Comment by LindaMcNulty** [comment]
Okay, I believe this is all set now.

**Comment by sonofmun** [comment]
Great!

**Pull request merged**